app/main.py

```diff
@@ -1,8 +1,33 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list: [Person] = []
+
+    for person in people:
+        person_list.append(
+            Person(
+                name=person["name"],
+                age=person["age"]
+            )
+        )
+
+    # Adding husband and wife for people
+    for person in people:
+        if person.get("husband") is not None:
```

Comment:
Is not None` is redundant here, since in your case if statement will be True if key exists